src/main/java/org/spongepowered/api/entity/living/animal/Horse.java

```diff
@@ -103,4 +103,26 @@
      */
     void setSaddle(@Nullable ItemStack itemStack);
 
+    /**
+     * Gets this horse's temper.
+     *
+     * <p>Temper defines how close a horse can become tamed by
+     * {@link org.spongepowered.api.entity.Tamer}s. The higher the temper,
+     * the more likely a tamer will tame the horse.</p>
+     *
+     * @return This horses current temper
+     */
+    int getTemper();
```

Comment:
To me it would seem like that a higher temper = angrier horse. Maybe a better name?